Core/Frameworks/Baikal/Core/Server.php

```diff
@@ -131,6 +132,10 @@ function start() {
      */
     protected function initServer() {
 
+        try {
+            $config = Yaml::parseFile(PROJECT_PATH_CONFIG . "config.yaml");
+        } catch (\Exception $e) {}
```

Comment:
Log the exception?

Reply:
Sure — do you mean `error_log()` for instance, or is there another helper that I should use ?

Reply:
_see my last commit_